public_html/layouts/basic/modules/Settings/Dav/resources/Dav.js

```diff
@@ -79,11 +79,26 @@ jQuery.Class('Settings_DAV_Js', {}, {
 				}
 		);
 	},
+	/**
+	 * Register buttons to copy password
+	 */
+	registerClipBoard: function () {
```

Comment:
The function should be universal and added in a file https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/public_html/layouts/resources/Fields.js